packages/network-controller/tests/provider-api-tests/shared-tests.ts

```diff
@@ -106,48 +116,51 @@ export const testsForProviderType = (providerType: ProviderType) => {
 
     describe('methods that assume there is no block param', () => {
       const assumingNoBlockParam = [
-        // { name: 'eth_getFilterLogs', numberOfParameters: 1 },
```

Comment:
Any idea why this one wasn't able to be tested with `web3-provider-engine`?

Reply:
@BelfordZ ^

Reply:
the filter & subscribe tests had some differences that were not deemed important to test since mobile implements its own subscribe / filter mw [in the background bridge](https://github.com/MetaMask/metamask-mobile/blob/main/app/core/BackgroundBridge/BackgroundBridge.js#L317), making it so that web3-provider-engines' implementation of these methods are not used. 

They wouldnt pass due to some difference in setTimeout use / requests that would actually be made, and since the code path was unused, they were just commented out.